src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -1,107 +1,107 @@
-	public void i_open_SED_accordion(String arg3) throws Throwable {
-		driver.get("https://chpl.ahrqstg.org/#/product/" + arg3);
-		listingDetails_page.sed_details_accordion_open(driver);
-	}
-	
-	@Then("^there should be text 'No Certification Criteria were tested for SED'$")
-	public void there_should_be_text_No_Certification_Criteria_were_tested_for_SED() throws Throwable {
-			
-		assertTrue(driver.getPageSource().contains("No Certification Criteria were tested for SED."));
-	}
-
-}
-
+package stepDefinitions;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import java.util.List;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import pageObjects.listingDetails_page;
+
+import static org.junit.Assert.assertTrue;
+
+public class SEDTweaksSteps {
+
+	public WebDriver driver;
+	
+	public SEDTweaksSteps()
+	{
+		driver = hooks.driver;
+		
+	}
+
+ 	
+	@Given("^I am on the Details page of Listing \"([^\"]*)\"$")
+	public void i_am_on_a_listing_s_details_pages(String arg1) throws Throwable {
+		driver.get("https://chpl.ahrqstg.org/#/product/" + arg1);
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		listingDetails_page.load_listings_details_page(driver); 	    
+	}
+
+	
+	@Then("^I should see the accordions in the following order: Criteria, CQMs, SED, G1/G2, Surveillance, Additional Info$")
+	public void i_should_see_the_accordions_in_the_following_order_Criteria_CQMs_SED_G1_G2_Surveillance_Additional_Info() throws Throwable {
+		
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		wait.until(ExpectedConditions.presenceOfElementLocated(By.id("details-certification-criteria")));
+		assertTrue(driver.findElement(By.id("details-certification-criteria")).getText().contains("Certification Criteria"));
+		assertTrue(driver.findElement(By.id("details-cqm")).getText().contains("Clinical Quality Measures"));
+		assertTrue(driver.findElement(By.id("details-sed")).getText().contains("Safety Enhanced Design (SED)"));
+		assertTrue(driver.findElement(By.id("details-g1g2")).getText().contains("Successfully Tested G1/G2 Measures"));
+		assertTrue(driver.findElement(By.id("details-surveillance-activities")).getText().contains("Surveillance Activities"));
+		assertTrue(driver.findElement(By.id("details-additional-information")).getText().contains("Additional Information"));
+		
+	}
+
+	@When("^I open SED details accordion$")
+	public void i_open_SED_accordion() throws Throwable {
+		listingDetails_page.sed_details_accordion_open(driver);
+	}
+	
+	@Then("^usability report text should read as 'Full Usability Report'$")
+	public void usability_report_title_reads_as_Full_Usability_Report() throws Throwable {
+		
+	assertTrue(driver.getPageSource().contains("Full Usability Report"));

```

Comment:
We want to use Page Objects to get to the items, right? So this `assert` should be something like `listingDetails_page.getSedUsabilityReportTitle().contains("Full Usability Report"));` and the Page Object should add that method (`getSedUsabilityReportTitle`) to match it